css/scss/pages/_lagar_regler_tillsyn.scss

```diff
@@ -0,0 +1,27 @@
+.bg-yellow {
+    background-color: yellow;
+    text-align: center;
+}
+.lrt-head {
+    font-size: 40px;
+    padding: 50px, 10px;
```

Comment:
This CSS is invalid, that's probably why you get an error 😉
If you have "watching" active, in Live Sass Compiler, it while saving the main style.scss it should give an error message, i.e. it won't let you save with incorrect CSS.